lms/djangoapps/class_dashboard/dashboard_data.py

```diff
@@ -87,6 +91,7 @@ def get_sequential_open_distrib(course_id):
 
```

Comment:
No, for now this just uses the analytics database and API via the client, but @jbau wants to make it parameterizable between the two data sources, so I'll be adding a parameter to switch shortly.